### DISCUSSION TITLE
Welcome to dart_beacon Discussions!

### DISCUSSION THREAD

**Discussion by zupat** [jinyus/dart_beacon#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arbile26** [comment]
Huge thanks for this masterpiece!
Hope it continues.
I'm planing to use it when it becomes stable.

**Reply by zupat** [reply]
There hasn't been any breaking changes for a while so it's close to being stable. I'll wait 2 more weeks before releasing 1.0
